pkg/bisect/bisect.go

```diff
@@ -254,14 +254,12 @@ func (env *env) bisect() (*Result, error) {
 	}
 	env.crashReport = testRes.rep
 
-	if len(cfg.Kernel.BaselineConfig) != 0 {
```

Comment:
It would be nice, if this was still disable-able somehow. Otherwise manually running syz-bisect locally (for development/debug purposes) becomes even more tedious.

Reply:
That particular baseline check was moved to `Minimize()`, so the question is more about 1 more run iteration to disable unnecessary instrumentation. I can add a new bisection config option to disable it, if you think it can be beneficial to disable it.

Reply:
Ah yes I see, hmm. Maybe don't bother then.

Background: Every time I needed to debug bisections, I wanted to get into the bisection logic itself. Before this PR there were already at least three run iterations before the git bisections starts (reproducible on original commit, reproducible on HEAD, reproducible on last kernel tag). That makes for quite an annoying turn around for every new try, however one more run is probably not going to change this. Maybe I'll add a way to mock all the initial runs for syz-bisect next time I need this.